image.go

```diff
@@ -2,31 +2,37 @@ package yoti
 
 import "encoding/base64"
 
-type ImageType int
+type imageType int
```

Comment:
Since you're exporting the values below I think you should export the type as well (and document it).